packages/core/src/external/aws/lambda-logic/bundle-to-brief.ts

```diff
@@ -308,21 +259,18 @@ function buildSlimmerPayload(bundle: Bundle): any[] | undefined {
     return [];
   });
 
-  // Filter out embedded resources from the final bundle
-  const filteredEntries = withFilteredReports?.flatMap(entry => {
-    if (Object.keys(entry).length === 0) return []; // TODO: Check why {} not being removed
-    if (referenceResources.includes(entry.resourceType)) return [];
-    if (containedResourceIds.includes(entry.id)) return [];
-    return {
-      ...entry,
-      id: undefined,
-    };
+  const filteredEntries: SlimResource[] = [];
+  withFilteredReports?.forEach(entry => {
```

Comment:
I'd either keep the comment or rename the const to give more meaning... "filtered" can mean multiple types of filtering done in the original list.

Reply:
I think I'll just factor this out into a function with a tsdoc